### PR TITLE
RSOP (Basic GPO Info)

### PR DESCRIPTION
**Pull request by ryanjoachim**
Bare minimum GPO information needed to be useful.

### CODE REVIEW

**Review by PDQDakota — DISMISSED** [review]

File: PowerShell Scanners/RSOP (Basic GPO Info)/Readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Instructions
+[How to use this repository](../../README.md)
+
+# Description
+
+Uses Get-CimInstance to pull basic GPO information on the target.
+Returns the following:
```
Comment (Colby-PDQ) [inline PowerShell Scanners/RSOP (Basic GPO Info)/Readme.md]: This won't format properly. Markdown requires either 2 spaces at the end of the previous line, or an empty line. https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#paragraphs

File: PowerShell Scanners/RSOP (Basic GPO Info)/Readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* Name (Name of the GPO)
+* Enabled (Is the GPO applied)
+* filterAllowed (Is the target restricted from applying the GPO)
```
Comment (Colby-PDQ) [inline PowerShell Scanners/RSOP (Basic GPO Info)/Readme.md]: `filterAllowed` should be `FilterAllowed`, to match the output of the script.

File: PowerShell Scanners/RSOP (Basic GPO Info)/Scan Profile.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<AdminArsenal.Export Code="PDQInventory" Name="PDQ Inventory" Version="19.0.40.0" MinimumVersion="19.0">
+  <ScanProfile>
+    <Collections type="list" />
+    <Scanners type="list">
+      <Scanner>
+        <ModifiedDate>2020-06-16T18:17:28.0000000-06:00</ModifiedDate>
+        <Name>RSPO (Basic GPO Info)</Name>
```
Comment (Colby-PDQ) [inline PowerShell Scanners/RSOP (Basic GPO Info)/Scan Profile.xml]: `RSPO` should be `RSOP`.

File: PowerShell Scanners/RSOP (Basic GPO Info)/Scan Profile.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        <Parameters></Parameters>
+        <AdditionalFiles></AdditionalFiles>
+        <RowLimit value="100" />
+        <TypeName>PowerShell</TypeName>
+        <SourceScannerId value="81" />
+      </Scanner>
+    </Scanners>
+    <Description></Description>
+    <ScanProfileId value="23" />
+    <Name>PS - RSOP (Basic GPO Info)</Name>
```
Comment (Colby-PDQ) [inline PowerShell Scanners/RSOP (Basic GPO Info)/Scan Profile.xml]: I recommend changing `Info` to `Information`, to match everything else.

**Review by Colby-PDQ — CHANGES_REQUESTED** [review]

**Review by ryanjoachim — COMMENTED** [review]
Changes (hopefully) made.

**Review by Colby-PDQ — APPROVED** [review]

**Pull request merged**